site/content/contribute/getting-started/code-review.md

```diff
@@ -79,10 +79,15 @@ If you are a core committer seeking a review
     * Note that the last core committer to approve your changes may do this on your behalf.
     * If your pull request depends on other pull requests, consider assigning the `Do Not Merge/Awaiting PR` label.
 10. Handle any cherry-picks.
```

Comment:
I would like to unify the two sources of truth. What do you think about merging the content from this file into `site/content/contribute/getting-started/branching.md` and link to it?

Reply:
@cpanato What do you think about this?

Reply:
done